tests/src/test/java/io/streamnative/pulsar/handlers/kop/KopProtocolHandlerTestBase.java

```diff
@@ -622,6 +625,12 @@ public void close() {
         }
     }
 
+    public static void addBrokerEntryMetadataInterceptors(ServiceConfiguration configuration) {
+        Set<String> interceptorNames = new HashSet<>();
+        interceptorNames.add("org.apache.pulsar.common.intercept.AppendBrokerTimestampMetadataInterceptor");
+        interceptorNames.add("org.apache.pulsar.common.intercept.AppendIndexMetadataInterceptor");
+        configuration.setBrokerEntryMetadataInterceptors(interceptorNames);
```

Comment:
Since after this PR KoP will require an extra config `brokerEntryMetadataInterceptors` for broker, it would be better to update the README: https://github.com/streamnative/kop#install-kop-protocol-handler